Lib/unittest/case.py

```diff
@@ -745,7 +745,11 @@ def __call__(self, *args, **kwds):
     def debug(self):
         """Run the test without collecting errors in a TestResult"""
         self.setUp()
-        getattr(self, self._testMethodName)()
+        testMethod = getattr(self, self._testMethodName)
+        if (not getattr(self.__class__, "__unittest_skip__", False) and
```

Comment:
I think it's not necessary use the round brackets for the condition.

Reply:
Do you mean the parens around the if statement? It is not necessary, but encouraged by the style guide to wrap multi-line if statements in parens: https://www.python.org/dev/peps/pep-0008/#multiline-if-statements

Reply:
Maybe it's out of the scope of this PR, but I'd like to suggest a small refactoring to make this condition a little bit clearer.

1. Define a new `_is_skipped(self, testMethod)` method:

```python
def _is_skipped(self, testMethod):
    return any(
        getattr(t, "__unittest_skip__", False)
        for t in (self.__class__, testMethod)
    )
```

2. Update the new condition to `not self._is_skipped(testMethod)` and the existing condition around line 665 to `self._is_skipped(testMethod)`

If it doesn't make sense to include this small update in this PR, but you think it makes sense in general, I'd be happy to work on it.